public/app/core/services/context_srv.ts

```diff
@@ -174,7 +174,7 @@ export class ContextSrv {
   }
 
   hasAccessInMetadata(action: string, object: WithAccessControlMetadata, fallBack: boolean) {
-    if (!config.rbacEnabled) {
```

Comment:
Q: What's the difference between `rbacEnabled` and `accessControlEnabled()`?

Reply:
There should be no difference. `accessControlEnabled` wraps `rbacEnabled`. Other methods use the method so I unified